### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by midzer** [lokal-ninja/ln-base#25]
We have problems

* in https://shoogle.net/overview/ page -> either move `<object>` to bottom or make it not visible to screen readers
* all other pages due inaccessible search combo boxes (combo box is emtpy after click) -> either hide for screen readers or find a way to make it accessible, i.e. give a hint that data is been loaded and you have to type something to filter for results

**Comment by midzer** [comment]
Done by https://github.com/lokal-ninja/ln-base/commit/e5392be0d89a44c3259027ab6d433a4054ef0bda and https://github.com/lokal-ninja/landingpage/commit/21f748a2e0388ef1200c21050ab8d3664c62cc4d
